File: Domain/Dto/Admin/GetAdminBusinessDetailsDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Dto.Admin
{
    public class GetAdminBusinessDetailsDto
    {
        public Guid Id { get; set; }
        public string BusinessName { get; set; }
        public string BusinessDesc { get; set; }
        public string BusinussPhone { get; set; }
        public string MangerName { get; set; }
        public string CityName { get; set; }
        public string AreaName { get; set; }
        public float BusinessRate { get; set; }

        public ICollection<string> Services { get; set; }
        public string BusinessPic { get; set; }
        public string LicencePic { get; set; }
    }
}

File: PetKeeperUI/src/app/_Models/GetAdminBusinessDetailsDto.ts
import { Guid } from "guid-typescript"

export class GetAdminBusinessDetailsDto {
    constructor(
        public id : Guid,
        public businessName:string,
        public businessDesc:string,
        public businussPhone:string,
        public mangerName:string,
        public cityName:string,
        public areaName:string,
        public businessRate:number,
        public services:string[],
        public businessPic:string,
        public licencePic:string,
    ) {}
}